### ISSUE TITLE
data-vocabulary.org schema deprecated

### ISSUE THREAD

**Issue by srcnsrcn** [amiyasahu/q2a-breadcrumbs#27]
i am getting data-vocabulary.org schema deprecated error

can you fix it?

**Comment by tryazi** [comment]
yes i am getting the same error. Will there be an update?

**Comment by tryazi** [comment]
@srcnsrcn bana email at sana dosyaları göndereyim. Sorunu çözdüm. infowebtesin@gmail.com

**Comment by amiyasahu** [comment]
I will take a look at this.

**Comment by tryazi** [comment]
I added the necessary changes, ami. we have a problem with meta position number. @amiyasahu

**Comment by srcnsrcn** [comment]
> I will take a look at this.

please asap

**Comment by srcnsrcn** [comment]
please fix it

**Comment by baoc75** [comment]
All set, except there just a problem in position meta @tryazi

**Comment by tryazi** [comment]
I will look at it when the job is gone

Thanks @baoc75

**Comment by mrzeeshanplus** [comment]
How you solved this ?

**Comment by baoc75** [comment]
@tryazi You are right, it's not necessary to declare meta position number as Google will understand its order.

**Comment by mrzeeshanplus** [comment]
Google is asking for 

position | A value for the position field is required.

https://search.google.com/structured-data/testing-tool/u/0/
see here @baoc75

**Comment by tryazi** [comment]
> Google is asking for
> 
> position | A value for the position field is required.
> 
> https://search.google.com/structured-data/testing-tool/u/0/
> see here @baoc75

Yeah. he wants it but solves the problem of giving them a fixed value

**Comment by tryazi** [comment]
> How you solved this ?

https://github.com/amiyasahu/q2a-breadcrumbs/pulls

Tryazi pulls good value

**Comment by mrzeeshanplus** [comment]
@tryazi should I put 
Content="1"

Can you please check this https://aptitude2quiz.com/33/

**Comment by tryazi** [comment]
> @tryazi should I put
> Content="1"
> 
> Can you please check this https://aptitude2quiz.com/33/

yes position always looks 1. you can replace it with js. but it's not a problem for google to stay like this. plugin maker and other developers did not solve. I think my solution is fine

please don't worry about it. then i can do a solution with js. but my work is very busy. I'm a farmer 😄

**Comment by mrzeeshanplus** [comment]
> > @tryazi should I put
> > Content="1"
> > Can you please check this https://aptitude2quiz.com/33/
> 
> yes position always looks 1. you can replace it with js. but it's not a problem for google to stay like this. plugin maker and other developers did not solve. I think my solution is fine
> 
> please don't worry about it. then i can do a solution with js. but my work is very busy. I'm a farmer

That's nice..... 
Also give your solution because I tried too many ways but only substituting content value as 1 works but that's not a perfect solution

**Comment by tryazi** [comment]
not perfect q2a. because developers no longer exist

**Comment by mrzeeshanplus** [comment]
> not perfect q2a. because developers no longer exist

But I think he still developing donut theme

**Comment by tryazi** [comment]
> > not perfect q2a. because developers no longer exist
> 
> But I think he still developing donut theme
> ""Latest commit by amiyasahu about 2 months ago"'"

it's now wordpress. the plug-in we already have problems with

**Comment by conceptworld** [comment]
> > How you solved this ?
> 
> https://github.com/amiyasahu/q2a-breadcrumbs/pulls
> 
> Tryazi pulls good value


Thanks a lot @tryazi . It works.

**Comment by baoc75** [comment]
Hi @conceptworld, I just set position=1 as the schema is [valid](https://validator.schema.org/) and the search result displays correct order of breadcrumb.

**Comment by conceptworld** [comment]
Thanks @baoc75 . This is solved the issue. And Google Search Console just sent a success email.
